README.md

```diff
@@ -18,7 +18,7 @@ Other platforms(including the Editor) have not been tested or/and may not work!
 
 ## Installation
 Copy this url: 
-```https://github.com/itisnajim/SocketIOUnity.git```
+```https://github.com/alex-suspicious/SocketIOUnityCallbacks```
```

Comment:
bring this back to https://github.com/itisnajim/SocketIOUnity.git, so i pull this request

Reply:
Done :)